test/optouts/add/OptOutsServiceAddTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Reflection;
+using com.esendex.sdk.optouts;
+using com.esendex.sdk.optouts.models;
+using com.esendex.sdk.optouts.models.response;
+using com.esendex.sdk.test.mockapi;
+using com.esendex.sdk.test.models.requests.optouts;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace com.esendex.sdk.test.optouts.add
+{
+    [TestFixture]
+    public class OptOutsServiceAddTests
+    {
+        private readonly Version _version = Assembly.GetAssembly(typeof (OptOutsService)).GetName().Version;
+        private mockapi.Request _request;
+        private OptOutCreateResult _result;
+        private string _phoneNumber;
+        private string _accountReference;
+        private string _expectedUserAgent;
+        private Guid _optOutId;
+        private DateTime _receivedAt;
+
+        [TestFixtureSetUp]
+        public void Given()
+        {
+            const string username = "user@example.com";
+            const string password = "heythiscantbeguessed";
+
+            _expectedUserAgent = string.Format("Esendex .NET SDK v{0}.{1}.{2}", _version.Major, _version.Minor, _version.Build);

```

Comment:
Do we have a User Agent stand alone test which doesn't use reflection - because this technically doesn't test anything beyond the first (none dynamic) element of the user agent?